worldedit-sponge/src/main/java/com/sk89q/worldedit/sponge/internal/SpongeWorldNativeAccess.java

```diff
@@ -105,8 +105,8 @@ public boolean updateTileEntity(BlockPos position, com.sk89q.jnbt.CompoundTag ta
         if (tileEntity == null) {
             return false;
         }
-        tileEntity.setLevelAndPosition(getWorld(), position);
```

Comment:
What is the point of calling `setLevelAndPosition` here?

Reply:
IIRC that data was missing from the BlockEntity instance

Reply:
> IIRC that data was missing from the BlockEntity instance

In this game version, position cannot be set, only level

Reply:
Is the position data available in the BlockEntity though? In the old version, that data was missing and had to be added